tests/avro/test_cached_client.py

```diff
@@ -140,19 +142,25 @@ def hash_func(self):
 
     def test_cert_no_key(self):
         with self.assertRaises(ValueError):
-            self.client = CachedSchemaRegistryClient(url='https://127.0.0.1:65534',
-                                                     cert_location='/path/to/cert')
+            self.client = CachedSchemaRegistryClient({
+                'url': 'https://127.0.0.1:65534',
+                'ssl.certificate.location': '/path/to/cert',
+            })
 
     def test_cert_with_key(self):
-        self.client = CachedSchemaRegistryClient(url='https://127.0.0.1:65534',
-                                                 cert_location='/path/to/cert',
-                                                 key_location='/path/to/key')
+        self.client = CachedSchemaRegistryClient({
+            'url': 'https://127.0.0.1:65534',
+            'ssl.certificate.location': '/path/to/cert',
+            'ssl.key.location': '/path/to/key'
+        })
         self.assertTupleEqual(('/path/to/cert', '/path/to/key'), self.client._session.cert)
 
-    def test_cert_path(self):
-        self.client = CachedSchemaRegistryClient(url='https://127.0.0.1:65534',
-                                                 ca_location='/path/to/ca')
-        self.assertEqual('/path/to/ca', self.client._session.verify)
+    def test_key_no_cert(self):
```

Comment:
Not sure what's being tested here, can you elaborate?

Reply:
^^